media-api/app/lib/ImageResponse.scala

```diff
@@ -24,7 +24,16 @@ object ImageResponse {
 
   def fileMetaDataUri(id: String) = URI.create(s"${Config.rootUri}/images/$id/fileMetadata")
 
-  def create(id: String, esSource: JsValue, withWritePermission: Boolean, included: List[String] = List()): (JsValue, List[Link]) = {
+  // TODO: move this as a method of Image (fiddly due to dep on Config)
+  def imageIsPersisted(image: Image) = {
```

Comment:
I like the idea of it being part of the companion object for the model, but if it's a pain, we'll address it when it's more of a pain not to have.